$OpenBSD: patch-wsrep-lib_src_CMakeLists_txt,v 1.1 2019/12/30 20:26:51 sthen Exp $

Stop libdl from being added to the linker command line.

Index: wsrep-lib/src/CMakeLists.txt
--- wsrep-lib/src/CMakeLists.txt.orig
+++ wsrep-lib/src/CMakeLists.txt
@@ -19,4 +19,4 @@ add_library(wsrep-lib
   transaction.cpp
   uuid.cpp
   wsrep_provider_v26.cpp)
-target_link_libraries(wsrep-lib wsrep_api_v26 pthread dl)
+target_link_libraries(wsrep-lib wsrep_api_v26 pthread ${LIBDL})
